buckets.yaml

```diff
@@ -31,3 +31,6 @@ gs://kubernetes-multiarch-e2e-results/logs/:
 gs://gce-daisy-test/logs/:
   contact: "crunk1"
   prefix: ""
+gs://istio-prow/:
+  contact: "chxchx"
```

Comment:
What does this mean?

Reply:
whoever to be the contact point for the bucket

Reply:
Is this name going to be included in email notification when failing or somewhere else?

Reply:
nope, I don't think this is used for anything else, @michelle192837 can confirm.

Reply:
this is just to document these buckets when someone else is wondering where these results are coming from :-)
this isn't in the email, the email contains some actual failure information